### PR TITLE
[nodes] KeyframeSelection: Add support for SfMData files as inputs and outputs

### PR DESCRIPTION
**Pull request by cbentejac**
## Description

This PR is related to https://github.com/alicevision/AliceVision/pull/1406, and adds the support of `SfMData` files as inputs and outputs.

In particular:
- It adds two new output parameters, `outputSfMDataKeyframes` and `outputSfMDataFrames` which are `SfMData` files which respectively contain all the selected keyframes and all the input frames that were not selected as keyframes;
- It updates the description of the input `inputPaths` parameter to indicate the `SfMData` files are supported;
- It updates the label of the `sensorDbPath` parameter (trivial).

## Features list

- [x] Update input's description to include `SfMData` files;
- [x] Add new output parameters for `SfMData` files.

### CODE REVIEW

File: meshroom/nodes/aliceVision/KeyframeSelection.py [hunk 1]
```diff
@@ -341,5 +341,20 @@ class KeyframeSelection(desc.AVCommandLineNode):
             value=desc.Node.internalFolder,
             uid=[],
         ),
+        desc.File(
+            name="outputSfMDataKeyframes",
+            label="Output SfMData Keyframes File",
```
Comment (mugulmd) [inline meshroom/nodes/aliceVision/KeyframeSelection.py:346]: The label seems a bit long compared to other similar labels in the UI. Maybe just `Keyframes SfMData` ?

File: meshroom/nodes/aliceVision/KeyframeSelection.py [hunk 2]
```diff
@@ -341,5 +341,20 @@ class KeyframeSelection(desc.AVCommandLineNode):
...
+            description="Output SfMData file containing all the selected keyframes.",
+            value=desc.Node.internalFolder + "keyframes.sfm",
+            uid=[],
+        ),
+        desc.File(
+            name="outputSfMDataFrames",
+            label="Output SfMData Frames File",
```
Comment (mugulmd) [inline meshroom/nodes/aliceVision/KeyframeSelection.py:353]: Same comment as for previous label

File: meshroom/nodes/aliceVision/KeyframeSelection.py
Comment (mugulmd) [inline meshroom/nodes/aliceVision/KeyframeSelection.py:1]: node version should be upgraded since this PR changes its attributes

Comment (cbentejac) [reply meshroom/nodes/aliceVision/KeyframeSelection.py:1]: Done.

**Review by mugulmd — APPROVED** [review]

**Pull request merged**